packages/backend/src/graphql/schema.graphql

```diff
@@ -803,6 +805,9 @@ type WalletAddress implements Model {
 
   "Additional properties associated with the wallet address."
   additionalProperties: [AdditionalProperty]
+
+  "Tenant ID of the wallet address."
+  tenantId: String
```

Comment:
I dont think we want to return the tenantId on the wallet address. unless im missing some use case?

Reply:
I think this can be used by the tenant operator during pagination, since then it'll be able to see which tenant this wallet address corresponds to. 

If we don't want to add this in the frontend, we can leave it out

Reply:
Agreed. Resolving this one.